lib/globus_client/endpoint.rb

```diff
@@ -10,10 +10,11 @@ class Endpoint # rubocop:disable Metrics/ClassLength
     # @param client [GlobusClient] a configured instance of the GlobusClient
     # @param path [String] the path to operate on
     # @param user_id [String] a Globus user ID (e.g., a @stanford.edu email address)
-    def initialize(client, path:, user_id:)
+    def initialize(client, path:, user_id:, notify_email: true)
```

Comment:
Can you add a new `@param` YARD doc above?

Reply:
good catch - added